src/styles/themes/b2c/variables.scss

```diff
@@ -151,23 +140,14 @@ $success: $color-special-success;
 // if necessary such Bootstrap generated styles need to be overridden in project styles (e.g. see buttons.scss)
 $primary: #006f6f;
```

Comment:
why is here the primary color defined - shouldn't be used here the color from the variable ?